src/identity/attestationMapping.tsx

```diff
@@ -0,0 +1,49 @@
+import { ReactNode } from 'react';
+import { Badge } from './components/Badge';
+import { getEASAttestations } from './getEASAttestations';
+import { Address, Chain } from 'viem';
+import { base } from 'viem/chains';
+
+type AttestationMethod = 'eas';
+
+type AttestationMapping = Record<
+  AttestationMethod,
+  {
+    Badge: ReactNode;
+    predicate: ({
+      address,
+      chain,
+      config,
+    }: {
+      address: Address;
+      chain: Chain;
+      config: unknown;
+    }) => boolean;
+  }
+>;
+
```

Comment:
Helping understand, why do we need a mapping? it feels more complexity to me.

What am I missing?

Reply:
We want to change two things depending on the attestation method:
1. Badge component
2. Business logic to determine if user is attested on a given method

Reply:
We could have something like: 
```
const mapping = {
  eas: {
    Badge: EasBadge,
    predicate: () => {...}
  },
  nouns: {
    Badge: NounsBadge,
    predicate: () => {...}
  }
}
```

I am assuming non-eas attestations will have different ways to check attestations of a given address

Reply:
@kyhyco just for context, EAS doesn't need this kind of mapping. As attestation are connected to a particular Wallet Address, and not to a namespace like ENS or Nouns.

So I have a good feeling this mapping is not necessary.

Reply:
deleted the mapping file